### PR TITLE
Add more detailed instructions for USB testing

### PR DESCRIPTION
**Pull request by kellysun878**
Signed-off-by: Kelly Sun <sunkelly888@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,19 +200,53 @@ webboot very much: It lists a kernel, an initrd, and extra arguments to append.
...
+1. Write a Volume Boot Record (VBR) to the stick
+2. Write a Master Boot Record (MBR) to it
+3. Mark the first partition as bootable
+4. Copy the config file, Linux kernel, and initcpio
+
+The following instructions will walk you through these four steps.
+Tip: You may need to replace `sdb1` with the name of your partition.
+
+To prepare your USB stick, run `sudo fdisk /dev/sdb` and use the fdisk instructions to complete the following:
+1. Delete all existing partitions
```
Comment (rjoleary) [inline README.md:221]: Add the letter you typed for each of these.

Comment (rjoleary) [reply README.md:221]: It might also help to include a paste of the output of fdisk.

File: README.md [hunk 2]
```diff
@@ -200,19 +200,53 @@ webboot very much: It lists a kernel, an initrd, and extra arguments to append.
 Before you continue, please make sure to meet the following conditions:
 
 - your system can boot from MBR (possibly through UEFI CSM)
-- you have a USB stick with MBR partitioning, first partition formatted as VFAT
-- you have a directory `/mnt/usb` to mount the partition to
-- you have `syslinux` installed (use your package manager)
-- you have built a suitable Linux kernel using a config from this repository
-- you have built an initramfs that includes webboot, gzip-compressed
+- You have a USB stick with MBR partitioning, first partition formatted as VFAT
```
Comment (rjoleary) [inline README.md:203]: These last four steps are now part of your instructions further down, so it is a bit redundant to list them here.

File: README.md [hunk 3]
```diff
@@ -200,19 +200,53 @@ webboot very much: It lists a kernel, an initrd, and extra arguments to append.
...
+- You have a directory `/mnt/usb` to mount the partition to
+- You have `syslinux` installed (use your package manager)
```
Comment (rjoleary) [inline README.md:205]: `sudo apt-get install syslinux`

File: README.md [hunk 4]
```diff
@@ -200,19 +200,53 @@ webboot very much: It lists a kernel, an initrd, and extra arguments to append.
...
+5. Save
+
+Set the partition type to VFAT
+```sh
+mkfs -t vfat /dev/sdb1
+```
+
+Install package libelf-dev
+```sh
+sudo apt install libelf-dev
```
Comment (rjoleary) [inline README.md:234]: Can this instead be in the prerequisites around line 125?

**Review by rjoleary — COMMENTED** [review]
I followed these instructions and they work :100:

**Review by rminnich — APPROVED** [review]

**Pull request merged**